stustapay/core/schema/order.py

```diff
@@ -3,20 +3,32 @@
 from typing import Optional
 from uuid import UUID
 
-from pydantic import BaseModel
+from pydantic import BaseModel, root_validator
 
 from stustapay.core.schema.product import Product
 
 
 class OrderType(enum.Enum):
     sale = "sale"
     topup_cash = "topup_cash"
-    topup_sum = "topup_sumup"
+    topup_sumup = "topup_sumup"
 
 
 class NewLineItem(BaseModel):
     product_id: int
-    quantity: int
+    quantity: Optional[int]
+    price: Optional[float]
```

Comment:
maybe write a comment that this is how free price items are requested for booking: product's fixed_price is False, and the requested price is then set here.